src/rpc/rawtransaction_util.h

```diff
@@ -5,16 +5,30 @@
 #ifndef BITCOIN_RPC_RAWTRANSACTION_UTIL_H
 #define BITCOIN_RPC_RAWTRANSACTION_UTIL_H
 
+#include <map>
+
 class CBasicKeyStore;
 class UniValue;
 struct CMutableTransaction;
+class Coin;
+class COutPoint;
 
 namespace interfaces {
```

Comment:
Remove.

Reply:
Seems I need to forward declare both Coin and COutPoint objects there ?

Reply:
I mean you can remove the interfaces namespace.

Reply:
promag is saying you can remove the forward declaration of the `interfaces::Chain` class.